NamedPipeProtocol.md

```diff
@@ -0,0 +1,15 @@
+# The GVFS Named Pipe Protocol (v1)

```

Comment:
I don't think we need to create a protocol doc for this. This feels like it could just be a class or method header comment up above.

This is not a standard protocol that we expect any other service or product to implement, and it doesn't even hold true across different versions of VFS for Git (meaning this protocol itself doesn't need to be versioned). This is just the internal details of how various processes in our codebase talk to each other.

Reply:
I agree - this wasn't meant to be versioned or for external components. My intention was to define the behavior in a single place for internal components. I will move this into a class comment - I think that would be a better fit.